example/app.config.js

```diff
@@ -0,0 +1,59 @@
+export default ({ config }) => ({
+  ...config,
+  name: 'example',
+  slug: 'example',
+  version: '0.0.1',
+  orientation: 'portrait',
+  icon: './assets/icon.png',
+  sdkVersion: '51.0.0',
+  splash: {
+    image: './assets/splash.png',
+    resizeMode: 'cover',
+    backgroundColor: '#F8F9FF',
+  },
+  ios: {
+    bundleIdentifier: 'com.example',
+    buildNumber: '1',
+    supportsTablet: true,
+  },
+  android: {
+    versionCode: 1,
+    package: 'com.example',
+    adaptiveIcon: {
+      foregroundImage: './assets/adaptive-icon.png',
+      backgroundColor: '#F8F9FF',
+    },
+  },
+  web: {
+    favicon: './assets/favicon.png',
+  },
+  plugins: [
+    [
+      'expo-camera',
+      {
+        cameraPermission: 'Allow RNGH example to access your camera',
+      },
+    ],
+    [
+      'expo-font',
+      {
+        fonts: [
+          './node_modules/@swmansion/icons/fonts/broken/swm-icons-broken.ttf',
+          './node_modules/@swmansion/icons/fonts/outline/swm-icons-outline.ttf',
+          './node_modules/@swmansion/icons/fonts/curved/swm-icons-curved.ttf',
+        ],
+      },
+    ],
+    [
+      'expo-build-properties',
+      {
+        ios: {
+          newArchEnabled: !!+process.env.FABRIC_ENABLED,
+        },
+        android: {
+          newArchEnabled: !!+process.env.FABRIC_ENABLED,
+        },
```

Comment:
WDYT @j-piasecki?

Reply:
I think that's essentailly `Number(process.env.FABRIC_ENABLED)` but less readable 😅

Reply:
It's a shorthand for `Boolean(Number(process.env.FABRIC_ENABLED))`, without `Boolean()` an error is thrown. 
Previously I wasn't sure if `!!+` was much of a problem, but now I agree the longer form is much more readable.
Fixed in [ae790b3](https://github.com/software-mansion/react-native-gesture-handler/pull/3204/commits/ae790b3673ed7f9016d6b963bbd9857dd606394b)